packages/tools/css-serializer/src/parse.ts

```diff
@@ -280,7 +281,11 @@ export function parse(content: string, options: {
           type: 'StyleRule',
           style: transformBlock(node.block, errors),
           selectorText: {
-            value: preludeText,
+            value: options.enableCSSSelector
+              ? preludeText
+              // When enableCSSSelector is disabled, remove backslash escapes to ensure CSS selector compatibility.
+              // For example, TailwindCSS generates selectors like `.h-\\[370px\\]` which require unescaping.
+              : preludeText.replace(/\\/g, ''),
```

Comment:
I don't think it is correct to use `.replace()` for all backslash.

It's valid to use selector with `\` in legacy CSS Selector:

```css
.Backg\\round {
}
```

<img width="946" alt="image" src="https://github.com/user-attachments/assets/a704971c-bd35-4a64-b583-5639a8a398c9" />

We should only escape the character that is meaningful in CSS. E.g.: `\[`, `\#`

Reply:
OK! Thanks for your comment! @colinaaa